style.css

```
@@ -0,0 +1,1708 @@
+
+    transform: rotate(-2deg);
+}
+
+    font-weight: bold;
+
+    text-shadow:
+        4px 4px 0 #111,
+        -1px -1px 0 #111;
+
+    white-space: nowrap;
+}
```

Stylelint (17.14.0)

[error] Unexpected } (CssSyntaxError)

(CssSyntaxError)